coverity-scan.sh

```diff
@@ -74,9 +88,11 @@ debugrun() {
 }
 
 scanit() {
-  export PATH="${PATH}:/opt/${COVERITY_BUILD_VERSION}/bin/"
+  progress "Scanning using coverity"
+  export PATH="${PATH}:${INSTALL_DIR}/${COVERITY_BUILD_VERSION}/bin/"
```

Comment:
I would prefer to avoid changing this. In bash is working fine to do `export PATH+="blah"`, but on zsh fails.
It won't be an issue given that we force bash, but probably better leave the more portable version just to be safe

Reply:
Well, zsh seems to work fine if you export in a separate step, but I agree let's leave it like this as it's the standard way for setting the path.